packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './delete-job.controller';
+import template from './delete-job.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    goToJobs: '<',
+    goToJobInfo: '<',
```

Comment:
Do you need both functions ? I think you could bind a single `goBack` function in the `.component` file, with its definition in the parent module.
For exemple, this is done in the confirm-delete modale in databases:
- [modale component file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/confirm-delete/confirm-delete.component.js#L7): here we add a `goBack` function in the bindings
- [parent 1 routing file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/databases.routing.js#L191): this is the first place where the modale can be displayed, we define a `goBack` function in the parent that will be used in the modale component
- [parent 2 routing file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/confirm-delete-database/confirm-delete-database.routing.js#L22): second place where the modale can be displayed

That way, your modale component can be used in any place, and should correctly redirect you to the parent location without a query param